app/controllers/stafftools/organizations_controller.rb

```diff
@@ -5,6 +5,21 @@ class OrganizationsController < StafftoolsController
 
     def show; end
 
+    def remove_user
+      not_found unless true_user.try(:staff?)
+
+      user = User.find(params[:user_id])
+
+      if Assignment.find_by(creator: user)
```

Comment:
I know that this will return nil if nothing is found, but would you mind adding a `.present?`